cli/tools/upgrade.rs

```diff
@@ -75,7 +76,8 @@ pub fn check_for_upgrades(cache_dir: PathBuf) {
       };
 
       let file = CheckVersionFile {
-        last_prompt: chrono::Utc::now(),
+        // put a date in the past here so that prompt can be shown on next run
+        last_prompt: chrono::Utc::now().sub(chrono::Duration::days(1)),
```

Comment:
We should probably use `UPGRADE_CHECK_INTERVAL + 1` with hours here to keep those linked or a minimum date/unix epoch.